test/unit/mwc-checkbox.test.js

```diff
@@ -41,11 +41,12 @@ test('element._formElement returns the native checkbox element', () => {
   assert.equal(element._formElement.localName, 'input');
 });
 
-test('element._component returns the mdc checkbox component', () => {
+test('element._component returns the mdc checkbox component', async () => {
+  await element.componentReady();
```

Comment:
This is the only functional change - I'm not sure if it's correct but other tests seemed to do this before reading `._component` and adding it here caused this test to stop failing.